src/components/SingleArticle.jsx

```diff
@@ -31,8 +36,21 @@ export default function SingleArticle() {
           <h1>{article.title}</h1>
           <img src={article.article_img_url} alt={article.title} />
           <p>{article.content}</p>
-          <p>Published on: {article.created_at}</p>
+          <FormatTime dateString={article.created_at} />{" "}
           <p>Votes: {article.votes}</p>
+          <div className="comments-section">
+            <h2>Comments</h2>
+            {comments && comments.length > 0 ? (
+              comments.map((comment) => (
+                <div key={comment.comment_id} className="comment">
+                  <h3>{comment.author}</h3>
+                  <p>{comment.body}</p>
+                </div>
+              ))
+            ) : (
+              <p>No comments to display.</p>
```

Comment:
Ideally we would want this message when there are no comments, at the moment this is displayed while comments are loading as well 🙂